.github/actions/build-keycloak/action.yml

```diff
@@ -46,7 +46,7 @@ runs:
       shell: bash
       run: |
         MVN_HTTP_CONFIG="-Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=120"
-        ./mvnw install -nsu -B -e -DskipTests -DskipExamples $MVN_HTTP_CONFIG 
+        ./mvnw install -nsu -B -e -DskipTests -DskipExamples $MVN_HTTP_CONFIG -Papp-server-wildfly
```

Comment:
We're actually planning on getting rid of `app-server-wildfly`, so that's not a solution as this would basically block our ability to remove the WildFly adapters.

Would it be a completely crazy idea to move adapter tests to `tests/adapter` leaving them as javax, with the embedded Undertow app server, but only supporting running with auth-server-quarkus rather than embedded auth-server?

Reply:
@stianst Using app-server-wildfly is just a temporary solution until we have a Quarkus 3 Alpha release with Hibernate 6. After that, we'll use app-server-undertow and auth-server-quarkus.

Reply:
Currently, we need to use Jakarta EE-based auth-server-undertow.

Reply:
Exactly as @vmuzikar mentioned.

Reply:
Not following. Are you saying we're not going to be able to run the auth-server-undertow anymore to run tests from within the IDE?

Reply:
We will be able to run general tests(not adapter tests) in IDE with auth-server-undertow. We need a different default app-server for the adapter tests (AFAIK, ATM, Undertow auth and app server run in the same JVM). I was thinking about the separation of these instances to different JVMs, and totally missed, that we have embedded Undertow :)) 

It'll be a better solution. Thanks.

Reply:
@vmuzikar Maybe it would be good to do it in a follow-up task as the fix for adapters itself is included here and confirmed it's working. WDYT?